ast/Trees.cc

```diff
@@ -686,10 +686,10 @@ string ConstantLit::showRaw(const core::GlobalState &gs, int tabs) {
     printTabs(buf, tabs + 1);
     fmt::format_to(std::back_inserter(buf), "orig = {}\n",
                    this->original ? this->original.showRaw(gs, tabs + 1) : "nullptr");
-    if (!resolutionScopes.empty()) {
+    if (resolutionScopes != nullptr && !resolutionScopes->empty()) {
```

Comment:
What do you think about ENFORCEing here that if `resolutionScopes` is not nullptr, that it's not empty?

Reply:
Works for me.

Reply:
Added. Also added to `_sanityCheck` on `ConstantLit`.